modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -1936,7 +1936,10 @@ def str_split(self, pat=None, n=-1, expand=False, regex=None):
     def unique(self, keep="first", ignore_index=True, subset=None):
         # kernels with 'pandas.Series.unique()' work faster
         can_use_unique_kernel = (
-            subset is None and ignore_index and len(self.columns) == 1 and keep
+            subset is None
+            and ignore_index
+            and len(self.columns) == 1
+            and keep is not False
```

Comment:
@dchigarev If we do not make this change, then the string is stored in the variable (for example, `"first"`). Did you do this on purpose?

Reply:
nope, it's a bug, thanks for the fix!